### PR TITLE
Fix/gh pages

### PR DESCRIPTION
**Pull request by emilioSp**
This PR fixes https://github.com/italia/api-oas-checker/issues/61 in the new-ui branch

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,24 +29,17 @@ spectral-security.yml: ./rules/  ./security/
...
-	git clone $(REPO_ORIGIN) $(TMPDIR) -b gh-pages
-	cp -r dist/* $(TMPDIR)
-	git -C $(TMPDIR) add .
-	git -C $(TMPDIR) -c user.name="gh-pages bot" -c user.email="gh-bot@example.it" \
-		commit -m "Script updating gh-pages from $(shell git rev-parse --short HEAD). [ci skip]"
-	git -C $(TMPDIR) push -q origin gh-pages
-	rm $(TMPDIR) -fr
\ No newline at end of file
+deploy: all
+	yarn deploy
```
Comment (bfabio) [inline Makefile:45]: We can probably get rid of this altogether and just call the deploy script from CircleCI when we are ready, [like in the developers.italia.it repo](https://github.com/italia/developers.italia.it/blob/master/.circleci/config.yml#L88).

Comment (emilioSp) [reply Makefile:45]: Yes! So I can remove the deploy section in this PR and then when we are ready I can script a GitHub Action to manage deploy and test. (I would prefer GitHub Action over CircleCI if it is possible)

@ioggstream do you agree?

...or we can get rid of the entire MakeFile if we extract the spectral ruleset generation in a separate script... @ioggstream ?

Comment (ioggstream) [reply Makefile:45]: WRT:
- gh-actions for gh-pages: if that works, it is ok
- circleci: we use it for tests, but if `italia/` is moving to gh-action it is fine, as long as test runs 
- Makefile: I like to type `make test`. Unless it is a problem, I'd retain it.

**Review by ioggstream — APPROVED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (emilioSp) [reply Makefile:45]: > gh-actions for gh-pages: if that works, it is ok
Yes it works, I have tried it on my forked [repo](https://emiliosp.github.io/api-oas-checker/)

**Pull request merged**